RadixWallet/Core/DesignSystem/Styles/SecondaryRectangularButtonStyle.swift

```diff
@@ -1,6 +1,7 @@
 // MARK: - SecondaryRectangularButtonStyle
 struct SecondaryRectangularButtonStyle: ButtonStyle {
 	@Environment(\.controlState) var controlState
+	var font: SwiftUI.Font = .app.body1Header
```

Comment:
```suggestion
	let font: SwiftUI.Font
```

since we already provide default on `init`